pkg/controller/migmigration/migrate.go

```diff
@@ -85,21 +82,25 @@ func (r *ReconcileMigMigration) ensureSourceClusterBackup(migMigration *migapi.M
 		}
 	}
 
-	vBackup := vrunner.BuildVeleroBackup(backupNsName, rres.migAssets.Spec.Namespaces, false)
+	vBackup := vrunner.BuildVeleroBackup(backupNsName, rres.MigAssets.Spec.Namespaces, false)
 	srcBackup, err := vrunner.RunBackup(srcClusterK8sClient, vBackup, backupNsName, logPrefix)
```

Comment:
Is this right? Won't we be calling `RunBackup` twice in this case? Once when you call `getSrcBackupName` and once again on `EnsureBackup`?

Reply:
Messed this up while rebasing... Good catch!